modules/custom/cu_advanced_content_bundle/cu_advanced_content_bundle.info

```diff
@@ -2,7 +2,7 @@ name = Advanced Content
 description = Add tabbed content, content lists, content grids and video reveal functionality to your site.
 core = 7.x
 package = CU-Advanced Content
-project_demo_url = http://www.colorado.edu/webcentral/node/668
+project_demo_url = https://websupport.colorado.edu/category/142-advanced-content
```

Comment:
I think the overview page is more useful than the category page for a bundle. Could we use links like https://websupport.colorado.edu/article/119-advanced-content when possible?

Reply:
I asked the Support Team this question. They said plz use the Categories, not Overview.

![Screen Shot 2019-08-16 at 1 49 37 PM](https://user-images.githubusercontent.com/638275/63194704-eafcad00-c02d-11e9-999f-075bccce20d5.png)